uxarray/grid/grid.py

```diff
@@ -1749,6 +1751,41 @@ def get_kd_tree(
 
         return self._kd_tree
 
+    def get_spatial_hash(
```

Comment:
Please include this in [`api.rst`](https://github.com/UXARRAY/uxarray/blob/main/docs/api.rst)

Reply:
I couldn't find an appropriate spot; `get_kd_tree` and `get_ball_tree` are not listed in there anywhere and these would be the most similar "search" methods..

Reply:
Oh that's a good catch; we need to include those.

Feel free to skip it for now, I'll open up a PR and include all three with appropriate sections after this is merged.

Reply:
Added to Grid/Methods section in api.rst in https://github.com/UXARRAY/uxarray/pull/1169/commits/b655a46cc9e699bbee43fd1b9561539d72bb2edd